apps/explorer/src/ui/SplitPanes.tsx

```diff
@@ -108,6 +122,7 @@ function SplitPanel({
             </Panel>
             {renderResizeHandle && (
                 <ResizeHandle
+                    noHoverHidden={noHoverHidden}
```

Comment:
It is used in 2 places, on the current implementation we hide the gutter when not hovered, but on the `module` page with code, it will always display there.